mars/opcodes.py

```diff
@@ -288,5 +288,8 @@
 RESHAPE_REDUCE = 2003
 DATAFRAME_INDEX_ALIGN_MAP = 2004
 DATAFRAME_INDEX_ALIGN_REDUCE = 2005
+SHUFFLE_MERGE = 2010
```

Comment:
Maybe call it with a prefix `DATAFRAME_` cuz it's used only for dataframe?

Reply:
Fixed. `DATAFRAME_` prefix has been added.